### PR TITLE
Add FormulatePro.app v0.0.6

### PR DESCRIPTION
**Pull request by shimeike**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Google code is going away soon. Are there no alternatives for this? If there are not, it isn’t worth including it.

**Comment by shimeike** [comment]
Ok, makes sense. There is a [macupdate download](http://www.macupdate.com/app/mac/19911/formulatepro) available is that acceptable? If so, I can update the cask with that, though there did not appear to be many existing casks using macupdate links ...
Looks like the original maintainer has ported the source to [github](https://github.com/adlr/formulatepro). What would the process be to get the dmg from google code added to github? (Sorry, I'm new to this ...)
Cheers!

**Comment by vitorgalvao** [comment]
> Looks like the original maintainer has ported the source to github

Great! Then it’s easy. I’ll comment on the lines.

File: Casks/formulatepro.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'formulatepro' do
+  version '0.0.6'
+  sha256 '9a5c37bad02a9dea7448e4ebe6fc6b0887efa05b0530603b9be1e5b0b3db2542'
+
+  url 'https://formulatepro.googlecode.com/files/FormulatePro-0.0.6.dmg'
+  name 'formulatepro'
+  homepage 'https://code.google.com/p/formulatepro/'
```
Comment (vitorgalvao) [inline Casks/formulatepro.rb]: Change to `https://github.com/adlr/formulatepro`.

**Comment by vitorgalvao** [comment]
> Ok, makes sense. There is a macupdate download available is that acceptable?

Not really, since it isn’t official. Best way to go about it is to ask the developers (perhaps on github) if they’ll be providing links to the built app.

**Comment by shimeike** [comment]
Currently no dmg on github ... will try to get one added.

**Comment by fanquake** [comment]
@shimeike Any luck getting a DMG added?

**Comment by shimeike** [comment]
No, sorry, haven't heard anything back from the developer. Happy to just forget about this if you'd rather not have it remain open indefinitely ...

Since the source is available, perhaps I should just work on a regular homebrew formula ... ? Any other thoughts or suggestions?

Cheers.

**Comment by vitorgalvao** [comment]
> Since the source is available, perhaps I should just work on a regular homebrew formula

Won’t the source be downloaded from google code as well? In that case the problem remains.

> Any other thoughts or suggestions?

You can fork it and serve it yourself, if you wish.

**Comment by shimeike** [comment]
>>Any other thoughts or suggestions?

>You can fork it and serve it yourself, if you wish.

Ok, thanks. I'll try that.

**Comment by vitorgalvao** [comment]
Alright. Will be waiting for that, then.

**Pull request closed without merging**